hr_employee_relative/data/data_relative_relation.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <!--
+          Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+          License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+        -->
+
+        <!-- hr.employee.relative.relation -->
+        <record model="hr.employee.relative.relation" id="relation_spouse">
```

Comment:
Odoo already has the field to set spouse information when the employee is married.

We could use that field when is the spouse?

Reply:
Please elaborate

Reply:
@luistorresm any news about this?

Reply:
The only thing I can think of is to compute that field depending if there's a valid relation, yet not sure if that totally OK, or that should go to a separate module

Reply:
I propose to avoid load the spouse data in this new feature,

And for that information use the Odoo fields,

What do you think?

Regards

Reply:
That would be 100% confusing for end-user. I'd hide the spouse UI field and set it to be related to data stored here in order to maintain compatibility with other modules that expect that field to be filled in with valuable data

Reply:
Actually, I gave that some thought - that particular feature should go to some `hr_employee_relatives_compat` or `hr_employee_relatives_spouse`